extensions/aris/src/bigDataCluster/dialog/addControllerDialog.ts

```diff
@@ -0,0 +1,493 @@
+			this.usernameInputBox.onTextChanged(e => this.toggleSignInButton());
+
+			usernameInputContainer.addItem(usernameInputText, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'width': '80px'
+				}
+			});
+
+			usernameInputContainer.addItem(this.usernameInputBox, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'margin': '-2px 0px 0px 0px',
+					'width': '100%',
+				}
+			});
+
+			allContainer.addItem(usernameInputContainer, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'margin': '-2px 0px 0px 0px',
+					'width': '100%'
+				}
+			});
+
+			let passwordInputContainer: azdata.FlexContainer = this.modelBuilder.flexContainer()
+				.withLayout(<azdata.FlexLayout>{
+					flexFlow: 'row',
+					alignItems: 'center',
+					width: '100%'
+				})
+				.component();
+
+			let passwordInputText = this.modelBuilder.text()
+				.withProperties<azdata.TextComponentProperties>({ value: 'Password' }).component();
+
+			this.passwordInputBox = this.modelBuilder.inputBox()
+				.withProperties<azdata.InputBoxProperties>({
+					placeHolder: 'password',
+					inputType: 'password',
+					value: this.prefilledValues.password
+				})
+				.component();
+			this.passwordInputBox.onTextChanged(e => this.toggleSignInButton());
+
+			passwordInputContainer.addItem(passwordInputText, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'width': '80px'
+				}
+			});
+
+			passwordInputContainer.addItem(this.passwordInputBox, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'margin': '-2px 0px 0px 0px',
+					'width': '100%',
+				}
+			});
+
+			allContainer.addItem(passwordInputContainer, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'margin': '-2px 0px 0px 0px',
+					'width': '100%'
+				}
+			});
+
+			this.rememberPwCheckBox = this.modelBuilder.checkBox()
+				.withProperties<azdata.CheckBoxProperties>({
+					label: 'Remember Password',
+					checked: this.prefilledValues.rememberPassword
+				}).component();
+			this.rememberPwCheckBox.enabled = false;
+
+			allContainer.addItem(this.rememberPwCheckBox, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					'justify-content': 'flex-start',
+					'align-items': 'center',
+					'margin': '0px 0px 0px 65px',
+				}
+			});
+
+			this.errorTextContainer = this.modelBuilder.flexContainer()
+				.withLayout(<azdata.FlexLayout>{
+					flexFlow: 'row',
+					alignItems: 'center',
+					width: '100%'
+				})
+				.component();
+
+			allContainer.addItem(this.errorTextContainer, {
+				flex: '1, 1, 0%',
+				CSSStyles: {
+					width: '100%'
+				}
+			});
+
+			tc.addContentContainer(allContainer);
```

Comment:
I wouldn't know what `tc` is here. Can we use a descriptive name (we discourage 1 or 2 letter names unless inside small lambda functions)

Reply:
fixed.